### ISSUE TITLE
iOS8 + Xcode6 performance issue 

### ISSUE THREAD

**Issue by tcamin** [BradLarson/GPUImage#1761]
I just wanted to check out if anybody is seeing weird performance issues using GPUImage on iOS8 when compiling with Xcode6 (GM).

Using Brad's FilterShowcase project (Objc-C) I compared the average framerate of the saturation filter on different devices compiled with Xcode 6 and Xcode 5.1. 

The only changes I made to the original project were the following:
1. enable -Ofast optimization
2. change the target's build configuration to RELEASE
3. set INITIALFRAMESTOIGNOREFORBENCHMARK to 60 to filter out initial performance noise

These are the average framerates I'm getting:

iPhone5S iOS8 (GM)
Xcode6:   1.29ms
Xcode5.1: 1.02ms

iPhone5C iOS8 (GM)
Xcode6:   2.02ms
Xcode5.1: 1.56ms

iPhone5 iOS7.0.2
Xcode6:   1.71ms
Xcode5.1: 1.74ms

I repeated the test and consistently getting the same results over and over again. All phones were put in airplain mode with no wifi and no bluetooth placing them flat down on a desk.

Basically iOS7 is seeing no performance penality while iOS8 has over 20% slow down when compiled with Xcode6.

I'm pretty sure it's something on my side, but would anybody care to repeat the above test and share the results?

Finally, if anybody is wondering, you can run (debug) apps on a iOS8 device using Xcode 5.1 (http://stackoverflow.com/a/24636679) but you cannot use instruments. Which makes it hard to see what's different between the 2 plattforms.
